### DISCUSSION TITLE
Allow tunnel exposure for certain services

### DISCUSSION THREAD

**Discussion by jaydrogers** [serversideup/spin#10] [Ideas]
# 👉 Describe the problem
[comment]: <> (What are you trying to solve?)
Developers need to be able to expose their development environments to the world for testing webhooks or getting feedback from Co-Workers.

# 👥 Problem evidence & reach
[comment]: <> (How many people have this problem?)
This would only affect a percentage of developers who need to use webhooks or ones who work in teams.

Laravel Sail is also doing this: https://github.com/laravel/sail/blob/1.x/bin/sail

# 🥰 Describe the "impact" on users?
[comment]: <> (How will this make people's lives better once it is solved?)
Developers would be pretty excited about this since they can share stuff without committing. It would also speed up development.

# 🏆 How to solve this problem
[comment]: <> (Describe the feature that you are proposing and how it will solve the problem)

Services to integrate with:
* https://expose.dev/
* https://ngrok.com/

# 💯 How do we validate the problem is solved?
[comment]: <> (Explain use cases on how we can measure the success of this implementation)
Developers should be able to easily share their work with a simple command (like `spin share`).

**Comment by voidintelligencetechnology** [comment]
This is gold dust! Wen `spin share`
